uECC.c

```diff
@@ -1233,6 +1233,7 @@ static int uECC_sign_with_k(const uint8_t *private_key,
                             const uint8_t *message_hash,
                             unsigned hash_size,
                             uECC_word_t *k,
+                            uint8_t *recid,
```

Comment:
Uh, that looks quite hacky. And `uECC_sign_with_k` is a static function, only used internally.